src/status_im/events.cljs

```diff
@@ -158,8 +158,10 @@
 
 (rf/defn on-going-in-background
   [{:keys [db now]}]
-  {:db         (assoc db :app-in-background-since now)
-   :dispatch-n [[:audio-recorder/on-background] [:audio-message/on-background]]})
+  {:db (assoc db :app-in-background-since now)
+   ;; event not implemented
```

Comment:
Because it is called with in a `on-going-in-background` method, so I thought I would leave it for when someone wants to call some method `on going in background` it would be very clear where to implement it.

Reply:
Cool @OmarBasem. Could you just add a missing semicolon there? There's a line with only one, but it's not a margin comment.

Reply:
if there's a clear task for this can't you link the corresponding issue too. you could also include this detail in the task description 👍

Reply:
@J-Son89 there is no task. I can remove this code completely, but as I said it would be helpful for when someone wants to call a method `on going in background`.

Reply:
if its usable why not :) its clojure not js :)